### DISCUSSION TITLE
Question, Is bao a separation kernel(SK)?

### DISCUSSION THREAD

**Discussion by wjhtinger** [bao-project/bao-hypervisor#23] [General]
What is the difference between the type1 hypervisor and separation kernel?

**Comment by josecm** [comment]
Very briefly, I would say that we could use a type-1 hypervisor to implement a separation kernel and that static partitioning hypervisors (such as bao) can indeed be considered a separation kernel implementation.

**Reply by wjhtinger** [reply]
> Very briefly, I would say that we could use a type-1 hypervisor to implement a separation kernel and that static partitioning hypervisors (such as bao) can indeed be considered a separation kernel implementation.

Thank your reply, and besides static partitioning hypervisors, is there any other ways to  implement a  separation kernel?
